luv 0.5.13 broke our Windows builds again

```
File "src/macro/eval/evalLuv.ml", line 1951, characters 29-37:
1951 | 			FS_event.start event path callback
                                    ^^^^^^^^
Error: This expression has type
         (string * Luv.FS_event.Event.t list, Luv.Error.t) Result.t -> unit
       but an expression was expected of type
         (string option * Luv.FS_event.Event.t list, Luv__.Error.t) result ->
         unit
       Type
         (string * Luv.FS_event.Event.t list, Luv.Error.t) Result.t =
           (string * Luv.FS_event.Event.t list, Luv.Error.t) result
       is not compatible with type
         (string option * Luv.FS_event.Event.t list, Luv__.Error.t) result 
       Type string is not compatible with type string option 
make: *** [Makefile:81: haxe] Error 1
```

--- a/haxe.opam
+++ b/haxe.opam
@@ -31,7 +31,7 @@ depends: [
   "conf-libpcre2-8"
   "conf-zlib"
   "conf-neko"
-  "luv" {= "0.5.12"}
+  "luv" {>= "0.5.13"}
   "ipaddr"
   "terminal_size"
 ]

--- a/src/macro/eval/evalLuv.ml
+++ b/src/macro/eval/evalLuv.ml
@@ -1942,7 +1942,7 @@ let fs_event_fields = [
 					) events
 				in
 				encode_obj [
-					key_file,vnative_string file;
+					key_file,encode_nullable vnative_string file;
 					key_events,encode_array vevents;
 				]
 			) v4
@@ -2175,7 +2175,7 @@ let env_fields = [
 let time_fields = [
 	"getTimeOfDay", vfun0 (fun() ->
 		encode_result (fun (t:Time.t) ->
-			encode_obj [key_sec,VInt64 t.tv_sec; key_usec,vint32 t.tv_usec]
+			encode_obj [key_sec,VInt64 t.sec; key_usec,vint32 t.usec]
 		) (Time.gettimeofday())
 	);
 	"hrTime", vfun0 (fun() ->
@@ -2292,10 +2292,10 @@ let resource_fields = [
 		encode_array_a [|vfloat m1; vfloat m5; vfloat m15|];
 	);
 	"freeMemory", vfun0 (fun() ->
-		VUInt64 (Resource.free_memory())
+		encode_nullable (fun u -> VUInt64 u) (Resource.free_memory())
 	);
 	"totalMemory", vfun0 (fun() ->
-		VUInt64 (Resource.total_memory())
+		encode_nullable (fun u -> VUInt64 u) (Resource.total_memory())
 	);
 	"constrainedMemory", vfun0 (fun() ->
 		encode_nullable (fun u -> VUInt64 u) (Resource.constrained_memory())

--- a/std/eval/luv/FsEvent.hx
+++ b/std/eval/luv/FsEvent.hx
@@ -26,7 +26,7 @@ enum abstract FsEventFlag(Int) {
 	/**
 		Starts the handle and watches the given path for changes.
 	**/
-	public function start(path:NativeString, ?flags:Array<FsEventFlag>, callback:(result:Result<{file:NativeString,events:Array<FsEventType>}>)->Void):Void;
+	public function start(path:NativeString, ?flags:Array<FsEventFlag>, callback:(result:Result<{file:Null<NativeString>,events:Array<FsEventType>}>)->Void):Void;
 
 	/**
 		Stops the handle.

--- a/std/eval/luv/Resource.hx
+++ b/std/eval/luv/Resource.hx
@@ -40,13 +40,15 @@ extern class Resource {
 
 	/**
 		Evaluates to the amount of free memory, in bytes.
+		Returns `null` when unknown.
 	**/
-	static function freeMemory():UInt64;
+	static function freeMemory():Null<UInt64>;
 
 	/**
 		Evaluates to the total amount of memory, in bytes.
+		Returns `null` when unknown.
 	**/
-	static function totalMemory():UInt64;
+	static function totalMemory():Null<UInt64>;
 
 	/**
 		Gets the amount of memory available to the process (in bytes) based on

